client/webserver/site/src/js/order.ts

```diff
@@ -258,33 +263,86 @@ export default class OrderPage extends BasePage {
     const tmpl = Doc.parseTemplate(matchCard)
     tmpl.status.textContent = OrderUtil.matchStatusString(m)
 
-    const setCoin = (pendingName: string, linkName: string, coin: Coin) => {
-      const formatCoinID = (cid: string) => {
-        if (cid.startsWith(coinIDTakerFoundMakerRedemption)) {
-          const makerAddr = cid.substring(coinIDTakerFoundMakerRedemption.length)
-          return intl.prep(intl.ID_TAKER_FOUND_MAKER_REDEMPTION, { makerAddr: makerAddr })
-        }
-        return cid
-      }
+    const tryShowCoin = (showPlaceholder: () => void, showCoin: () => void, linkName: string, coin: Coin) => {
```

Comment:
Applied, however I'd also suggest minor improvement on top: https://github.com/decred/dcrdex/pull/2042/commits/17542c2c5a6995bf8d62183c17ee776a24ec33f8 to keep all the code related to `refund` together in one place, instead of spread out (with that extra `if (m.refund) {`), thoughts ?

**update:** it should be also `if (m.refund) {` -> `if (!m.refund) {` in your example, right ? I noticed that only after I've done that refactoring https://github.com/decred/dcrdex/pull/2042/commits/17542c2c5a6995bf8d62183c17ee776a24ec33f8 (pulling this code close together)

Reply:
Applied, however I'd also suggest minor improvement on top: https://github.com/decred/dcrdex/pull/2042/commits/17542c2c5a6995bf8d62183c17ee776a24ec33f8 to keep all the code related to `refund` together in one place, instead of spread out (with that extra `if (m.refund) {`), thoughts ?

**update:** it should be also `if (m.refund) {` -> `if (!m.refund) {` in your example, right ? I noticed that only after I've done that refactoring https://github.com/decred/dcrdex/pull/2042/commits/17542c2c5a6995bf8d62183c17ee776a24ec33f8 (pulling this code close together)